index.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/d86848cfe0.js" crossorigin="anonymus"></script>
+    <link href="https//fonts.googleapis.com/css2? family=r=Roboto:wght@100;400;500;700;&display=swap" rel="stylesheet">
+    <title>Youtube Clone</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div id="logo">
+                <div id="hamburger">
+                    <a href="#"><i class="fa fa-bars"></i></a>
+                </div>
+                <a href="#"><img src="./image/YouTube-Logo.png  " width="80" height="40" alt="Youtube Logo"></a>
```

Comment:
[OPTIONAL]

Remember to not use inline styling and do not leave unused code in your codebase.

Reply:
Thank you for your feedback